### ISSUE TITLE
The color contrast of focus on each item under session details drop down is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#125917]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11YWCAG2.1;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG1.4.11;#A11ySev2;#CAITestPass_May2020;#Visual Studio Code;#ColorContrast;

### Repro Steps:

1. Launch VS Code
2. Navigate to the “sign-in” button using tab .
3. Tab to share button and use enter key.
4. Tab to Session Details dropdown and press enter key to expand it .
5. Use tab key to move to the tree view and then down arrow key to each item.
6. Launch accessibility insights for desktop and select the third tab contrast 

### Actual:
On checking the color contrast ratio of the items that has navigational focus it is below 3:1 is it as 1.101:1

### Same issue observed in the following screens:

1. no collaboration Planned Sessions
2. no collaboration Contacts
3. with collaboration Session Details
4. with collaboration  Contacts
5. with collaboration help
6. Audio Call
7. Contacts-Change Your Own availability Status
8. Contacts-Collaborate with  a recent contact
9. Contacts-Collaborate with suggested contact

### Expected:
The ratio of focus on the items while navigating to each should be equal or grater than 3:1

### User Impact:
Low vision users will not be able to view the focus on the items they are navigating to

**Comment by setaskin** [comment]
@isidorn ,  this is a bug that is originally assigned to Live Share team but since we have no custom color control over this , this issue is created over VS Code side.

**Comment by isidorn** [comment]
@setaskin can you just please attach a screen shot so know what the user is talking about. Thanks

**Comment by setaskin** [comment]
The author of the issue is from our test team.  Sure here is the screen shot  ![1214901_BM_VisualStudioCodeLiveShare-Sep2020_Nocollaboration_SessionDetails_ColorContrast](https://user-images.githubusercontent.com/37163639/121604132-da85a580-c9fe-11eb-9fdc-5bb8e0d5c45a.png)

**Comment by isidorn** [comment]
So this is the list focus color if I am not mistaken.

This should be fixed in latest insiders - we are no longer using just a background color but an outline.
Can you please try it out with Insiders? Thank you

fyi @sana-ajani 

![Screenshot 2021-06-11 at 11 18 22](https://user-images.githubusercontent.com/1926584/121663475-be3d4380-caa6-11eb-9925-4969a79df2c7.png)

**Comment by setaskin** [comment]
thanks @isidorn.  @fsteffi , would you double check if it is still a repro?

**Comment by fsteffi** [comment]
@setaskin  the issue still reproduces

**Comment by isidorn** [comment]
@fsteffi can you please add a picture from latest insiders, thanks

**Comment by fsteffi** [comment]
@isidorn  the image is provided in the bug @setaskin  can you please share the image

**Comment by isidorn** [comment]
The contrast is only not good when the dom focus is not in the list / tree.
So when the user has focus inside the list and tree it is ok. If that is not the case please let me know. @setaskin 

This is not important -> backlog.

**Comment by isidorn** [comment]
I have unlocked the issue. Sorry about that, the bot sometimes auto locks things.
